src/main/java/seedu/address/model/session/SessionType.java

```diff
@@ -0,0 +1,24 @@
+package seedu.address.model.session;
+
+/**
+ * Represents a session type. Session types follows the same specifications as the TSS.
+ * Example session types include: Tutorial, Grading, Consultation, etc.
+ */
+public enum SessionType {
```

Comment:
Good thinking making this into an enum class! We might need to do the same for the other enums